pkg/satellite/stream/stream.go

```diff
@@ -76,8 +76,7 @@ type satelliteStream struct {
 	recvChan           chan<- []byte
 	recvLoopClosedChan chan struct{}
 
-	latestByteTime  time.Time
-	closeTickerChan chan struct{}
+	latestByteTime time.Time
```

Comment:
I cannot find any references to this struct field anywhere in the code. Has it become obsolete?

Reply:
Good catch. This is indeed no longer used.